.github/ISSUE_TEMPLATE/cli-issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: CLI Issue
+about: Use this to report an issue with the CLI.
+title: ''
+labels: CLI
+---
+
+### Describe the issue
+<!-- A clear and concise description of what the issue is -->
+
+### Steps to reproduce the behavior
+<!--  Please list the steps required to reproduce the issue, for example:
+1. Run `databricks clusters ...`
+2. See error -->
```

Comment:
Why are these comment blocks?

Reply:
If a user does not fill these sections out, having them as comments will not clutter the issue. Also this is what we do in the terraform repo, and it works well enough there. 

On the other hand, having these as plain text incentivises filling them out. I'm fine with either approach. Uncommented the blocks for now, we can always revert this.